bundle/manifests/security-profiles-operator.x-k8s.io_rawselinuxprofiles.yaml

```diff
@@ -123,7 +123,6 @@ spec:
                       type: string
                   required:
                   - lastTransitionTime
-                  - message
```

Comment:
Those fields are auto-generated. I think we have to change the go code to fix that up.

Reply:
Is there a way to override the "+required"?
https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Condition

I'm afraid the fix might mean reverting to using self defined `Condition`.